app/helpers/state_file/nj_homeowner_eligibility_helper.rb

```diff
@@ -1,23 +1,22 @@
 module StateFile
   module NjHomeownerEligibilityHelper
     INELIGIBLE = :ineligible
-    UNSUPPORTED = :unsupported
+    WORKSHEET = :worksheet
     ADVANCE = :advance
 
     class << self
       def determine_eligibility(intake)
         is_ineligible = intake.homeowner_home_subject_to_property_taxes_no? ||
-          intake.household_rent_own_neither? ||
-          (intake.homeowner_main_home_multi_unit_yes? && intake.homeowner_main_home_multi_unit_max_four_one_commercial_no?)
-        is_unsupported = intake.homeowner_more_than_one_main_home_in_nj_yes? ||
-          intake.homeowner_shared_ownership_not_spouse_yes? ||
-          intake.household_rent_own_both? ||
-          (intake.homeowner_main_home_multi_unit_yes? && intake.homeowner_main_home_multi_unit_max_four_one_commercial_yes?)
+                        intake.household_rent_own_neither? ||
+                        (intake.homeowner_main_home_multi_unit_yes? && intake.homeowner_main_home_multi_unit_max_four_one_commercial_no?)
+        requires_worksheet = intake.homeowner_more_than_one_main_home_in_nj_yes? ||
+                             intake.homeowner_shared_ownership_not_spouse_yes? ||
+                             (intake.homeowner_main_home_multi_unit_yes? && intake.homeowner_main_home_multi_unit_max_four_one_commercial_yes?)
 
         if is_ineligible
           INELIGIBLE
-        elsif is_unsupported
-          UNSUPPORTED
```

Comment:
[dust] - i know i'm late to this review since i was out, so feel free to decline this. But ever since I introduced the `NjPropertyTaxEligibility` util, I've wondered if `NjHomeownerEligibilityHelper` and `NjTenantEligibilityHelper` should be combined into it. They are really all about Property Tax Eligibility, and just checking different criteria. It now feels like having three separate utils is actually more confusing. What do you think of combining them, so that there's just one place to check if (slash if we know if) the taxpayer is eligible for the property tax deduction or credit?

Reply:
Hey Matt, appreciate you taking a look and leaving comments.

I agree with you conceptually that it's probably easier to coalesce all these eligibility checks into `NjPropertyTaxEligibility`. Just for the sake of getting this merged by EOD today though, I'm going to leave it out of scope of this PR. Happy to do this refactor on one of the next PRs that touches homeowner/tenant eligibility.

Reply:
Makes sense to me 👍